### DISCUSSION TITLE
Select Main Object from Image

### DISCUSSION THREAD

**Discussion by firaki12345-cmd** [ImageMagick/ImageMagick#4344] [Help]
**Hi is there a way to select Main object from image and then trim it and save it as jpeg ?** 

i have several shoes Pics and want to optimize it for the web.

**Example Image:**

https://i.postimg.cc/GmTj8QFb/27.jpg

![27](https://user-images.githubusercontent.com/77625215/136658820-31cc644a-4f97-430b-8487-5b45965bda35.jpg)

**The Result i want After Selecting and Then Trimming it:**

https://i.postimg.cc/Kj95Lpp8/28.png

![28](https://user-images.githubusercontent.com/77625215/136658837-96be2dcd-8b2f-42ff-a372-610610183bf9.png)

**Thanks** :)

**Comment by snibgo** [comment]
Have you tried the "-trim" operation?
